pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -214,6 +246,13 @@ pub mod pallet {
 		RelayerMessageDecodingFailed { reason: RelayerMessageDecodingError },
 	}
 
+	#[pallet::hooks]
+	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
+		fn on_idle(_now: T::BlockNumber, max_weight: Weight) -> Weight {
+			Self::service_outbound_message_queue(max_weight)
+		}
```

Comment:
Additionally I would add an extrinsic to progress a specific message manually inc ases the chain is busy.

Reply:
Added an extrinsic for that.